### PR TITLE
Moving from React to Preact

### PR DESCRIPTION
**Pull request by pdt1806**
I hereby announce that this Preact version of vSuS landing page, which is 88% ~~thịt~~ identical to the old React version, is now able to be served to users. Performance is no longer an issue, and it never is (since this website has nothing much lol).

### CODE REVIEW

**Comment by tobycm** [comment]
> Performance is no longer an issue, and it never is

Load time bro (i mean a whole 7MB video in background...)

**Comment by pdt1806** [comment]
> > Performance is no longer an issue, and it never is
> 
> Load time bro (i mean a whole 7MB video in background...)

oh damn riel riel
hail toby for his great contribution to optimizing the bg video

File: src/pages/404/Error404.tsx [hunk 1]
```diff
@@ -1,44 +1,30 @@
...
-  }));
-  
-  export default function Error404() {
-    const { classes } = useStyles();
-  
-    useEffect(() => {
-      document.title = "404 Not found - vSuS";
-    }, []);
+import { Container, Image, Text, Title } from "@mantine/core";
+import { useEffect } from "react";
```
Comment (tobycm) [inline src/pages/404/Error404.tsx:2]: `react` -> `preact/hooks` but eh it's fine...

**Review by tobycm — APPROVED** [review]
Nice preact and refactoring siuuuuuuuuuuuu

**Pull request merged**